src/runtime/virtcontainers/api.go

```diff
@@ -138,766 +132,7 @@ func createSandboxFromConfig(ctx context.Context, sandboxConfig SandboxConfig, f
 	return s, nil
 }
 
-// DeleteSandbox is the virtcontainers sandbox deletion entry point.
-// DeleteSandbox will stop an already running container and then delete it.
-func DeleteSandbox(ctx context.Context, sandboxID string) (VCSandbox, error) {
-	span, ctx := trace(ctx, "DeleteSandbox")
-	defer span.Finish()
-
-	if sandboxID == "" {
-		return nil, vcTypes.ErrNeedSandboxID
-	}
-
-	unlock, err := rwLockSandbox(sandboxID)
-	if err != nil {
-		return nil, err
-	}
-	defer unlock()
-
-	// Fetch the sandbox from storage and create it.
-	s, err := fetchSandbox(ctx, sandboxID)
-	if err != nil {
-		return nil, err
-	}
-
-	// Delete it.
-	if err := s.Delete(); err != nil {
-		return nil, err
-	}
-
-	return s, nil
-}
-
-// FetchSandbox is the virtcontainers sandbox fetching entry point.
-// FetchSandbox will find out and connect to an existing sandbox and
-// return the sandbox structure. The caller is responsible of calling
-// VCSandbox.Release() after done with it.
-func FetchSandbox(ctx context.Context, sandboxID string) (VCSandbox, error) {
```

Comment:
the diagram https://github.com/kata-containers/kata-containers/blob/main/docs/design/kata-api-design.md#sandbox-connection-plugin-workflow still shows using this API to get sandbox, better to update it.